.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,154 @@
+      - name: Check build status in core
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          REPO: open-telemetry/opentelemetry-collector
+        run: |
+          RESULT=$(gh run list --branch main --json status --jq '[.[] | select(.status != "queued" and .status != "in_progress")][0].status' --workflow build-and-test --repo ${REPO} )
+          if [ "${RESULT}" != "completed" ]; then
+              echo "Build status in ${REPO} is not completed: ${RESULT}"
+              echo "$(gh run list --branch main --json status,url --jq '[.[] | select(.status != "queued" and .status != "in_progress")][0].url' --workflow build-and-test --repo ${REPO} )"
+              exit 1
+          fi
+      # Make sure the current main branch build successfully passes (Contrib).
+      - name: Check build status in contrib
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          REPO: open-telemetry/opentelemetry-collector-contrib
+        run: |
+          RESULT=$(gh run list --branch main --json status --jq '[.[] | select(.status != "queued" and .status != "in_progress")][0].status' --workflow build-and-test --repo ${REPO} )
+          if [ "${RESULT}" != "completed" ]; then
+              echo "Build status in ${REPO} is not completed: ${RESULT}"
+              echo "$(gh run list --branch main --json status,url --jq '[.[] | select(.status != "queued" and .status != "in_progress")][0].url' --workflow build-and-test --repo ${REPO} )"
+              exit 1
+          fi
+      # Update Contrib to use the latest in development version of Core. Run make update-otel in
+      # Contrib root directory and if it results in any changes, submit a PR. Open this PR as draft.
+      # This is to ensure that the latest core does not break contrib in any way. We’ll update it
+      # once more to the final release number later.
+      - name: Make update-otel in contrib
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          REPO: open-telemetry/opentelemetry-collector-contrib
+        run: "echo TODO" # TODO: This may require a webhook in contrib
+      - uses: actions/checkout@v3
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+      # Prepare Core for release.
+      #   - Update CHANGELOG.md file, this is done via chloggen
+      #   - Run make prepare-release PREVIOUS_VERSION=1.0.0 RELEASE_CANDIDATE=1.1.0 MODSET=stable
+      #   - Run make prepare-release PREVIOUS_VERSION=0.52.0 RELEASE_CANDIDATE=0.53.0 MODSET=beta
+      - name: Prepare release for core
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          REPO: open-telemetry/opentelemetry-collector
+          CANDIDATE_BETA: ${{ inputs.candidate-beta }}
+          CANDIDATE_STABLE: ${{ inputs.candidate-stable }}
+          CURRENT_BETA: ${{ inputs.current-beta }}
+          CURRENT_STABLE: ${{ inputs.current-stable }}
+        run: |
+          make chlog-install
```

Comment:
With the changes made to tooling, you shouldn't need to run `*install` anymore. Running the commands should check if the required command is built and build as required.

Reply:
This has been removed.